roles/add_kernel_parameters/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2024 RTE
+# Copyright (C) 2024 Savoir-faire Linux, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+
+- name: Check if we need to mount the boot partition
+  stat:
+    path: /usr/share/update/mount_boot.sh
+  register: mount_boot_script
+
+- name: Mount the boot partition if needed
+  command: /usr/share/update/mount_boot.sh
+  when: mount_boot_script.stat.exists
+
+- name: Detect the grub update command
+  stat:
+    path: /usr/sbin/update-grub
+  register: grub_update_command
+  when: grub_update_command is not defined
+
+- name: Set the grub update command to "update-grub"
+  set_fact:
+    grub_update_command: "update-grub"
+  when:
+    - grub_update_command is not defined
+    - not grub_update_command.stat.exists
```

Comment:
Why ?
if grub_update_command.stat.exists is false, it means that "update-grub" binary doesn't exists